planner/core/testdata/plan_suite_out.json

```diff
@@ -6166,9 +6167,9 @@
         "SQL": "SELECT ta.NAME,(SELECT sum(tb.CODE) FROM tb WHERE ta.id = tb.id) tb_sum_code FROM ta WHERE ta.NAME LIKE 'chad999%'",
         "Plan": [
           "HashJoin 250.00 root  left outer join, equal:[eq(test.ta.id, test.tb.id)]",
-          "├─IndexLookUp(Build) 250.00 root  ",
```

Comment:
Expected, IndexLookup cost is under-estimated in model1, so model2 prefers to use Scan instead of Lookup if no much data to access to reduce risks.